<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  border: 1px solid black;
  background-color: #0000ff;
  color: white;
  border-radius: 5px;
  outline: none;
  padding: 5px 10px;
}</string>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  border: 1px solid black;
  background-color: #0000ff;
  color: white;
  border-radius: 5px;
  outline: none;
  padding: 5px 10px;
}</string>
   </property>
   <property name="text">
    <string>Delete Table</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dropDown">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
